understanding/22/target-size-minimum.html

```diff
@@ -52,9 +52,9 @@ <h2>Intent of Target Size (Minimum)</h2>
 
         <ol>
-            <li>If a target is smaller than 24 by 24 CSS pixels but has sufficient spacing, it passes. The spacing is sufficient if the offset between targets is at least 24 CSS pixels. Target offset measures the distance between targets, measured from the farthest point of one target to the nearest point of the adjacent target, and is assessed for each of the adjacent targets. What edge of the target is <em>farthest</em> depends on the relative position of the neighboring target. For example, if considering distance above the target, the farthest edge would be the bottom of the target, and the offset would be measured from this bottom edge to the bottom edge of the neighboring target above. Note that where the targets differ in size or shape, the offset between targets may also differ. The offset distance must be at least 24 CSS pixels in all cases to fall under this exception.</li>
-            <li>The Success Criterion does not apply to inline targets in blocks of text – for example, text links inside a sentence or paragraph. This exception is allowed because text reflow based on viewport size makes it impossible for authors to anticipate where links may be positioned relative to one another. When multiple links are embedded in blocks of texts in smaller text sizes, maintaining a 24 CSS pixels offset between links in adjacent lines of text would require a large line height which can be undesirable in many design contexts. Also, inline numeric footnotes common in scientific texts may sometimes have a width well below 24 CSS pixels. Note: Stacked lists of links, as in navigation structures, do not count as inline links. Authors can anticipate the relative position of these links and accommodate sufficient target spacing.</li>
-            <li>If the spacing of the targets is essential to the information being conveyed, the "Essential" exception applies. For example, in digital maps, the position of pins is analogous to the position of places shown on the map. If there are many pins close together, the offset between pins and neighboring pins will often be below 24 CSS pixels. It is essential to show the pins at the correct map location, therefore the Essential exception applies. As well, some jurisdictions legally require online forms to replicate paper forms, which can impose constraints on the size of targets. In such jurisdictions, any legal requirement to replicate small targets can be considered essential.</li>
+            <li>If a target is smaller than 24 by 24 CSS pixels but has sufficient spacing, it can pass. The spacing is sufficient if there is a 24 CSS pixel circle that you can draw without overlapping another target. A point in the target that has 12 pixels in each direction creates that circle. For example, if a target is 16 CSS pixels wide, but there is no other target within 4 pixels, it would pass.</li>
```

Comment:
```suggestion
            <li>If a target is smaller than 24 by 24 CSS pixels but has sufficient spacing, it can pass. The spacing is sufficient if there is a 24 CSS pixel diameter circle that you can draw without overlapping another target. A point in the target that has 12 pixels in each direction creates that circle. For example, if a target is 16 CSS pixels wide, but there is no other target within 4 pixels, it would pass.</li>
```